mayastor-test/e2e/common/util.go

```diff
@@ -90,6 +94,22 @@ func GetMSV(uuid string) *MayastorVolStatus {
 				}
 				Expect(msVol.Node != "")
 				break
+			case "replicas":
+				replicas := val.Interface().([]interface{})
+				for _, replica := range replicas {
+					replicaMap := reflect.ValueOf(replica)
+					if replicaMap.Kind() == reflect.Map {
+						for _, field := range replicaMap.MapKeys() {
+							switch field.Interface().(string) {
+							case "node":
+								value := replicaMap.MapIndex(field)
+								msVol.Replicas = append(msVol.Replicas, value.Interface().(string))
+								break
```

Comment:
Is this break really needed? It breaks out of innermost `switch` which looks like C, but isn't needed in golang AFAIK: https://golang.org/ref/spec#Break_statements

Reply:
You are right, I'll fix that.